cmd/loadtest/terraform.go

```diff
@@ -26,13 +26,42 @@ func RunDeployCmdF(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func MakeDeployCommand() *cobra.Command {
+func RunDestroyCmdF(cmd *cobra.Command, args []string) error {
+	config, err := loadtest.GetConfig()
+	if err != nil {
+		mlog.Error(err.Error())
+		return nil
```

Comment:
any reason not to return `err` ?

Reply:
Yes, this is a problem in other places too. The issue with returning an error from a `cobra.Command` is that cobra thinks that there was an issue with the command flag or some incorrect param, and it spits out the entire command help section, which looks a bit odd to me.

It would give something like:
```
INFO[2020-02-05T17:52:18.200545+05:30]                                       caller="terraform/create.go:107"
ERRO[2020-02-05T17:52:18.200576+05:30] exit status 1                                 caller="loadtest/terraform.go:24"
Error: exit status 1
Usage:
  loadtest env create [flags]

Flags:
  -h, --help   help for create

Global Flags:
  -c, --config string   path to the configuration file to use

exit status 1
```

If we return nil, this help section does not get shown.

Reply:
Yuck. Thanks for clarifying